cmd/kind/load/docker-image/docker-image.go

```diff
@@ -116,8 +116,8 @@ func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
 	// pick only the nodes that don't have the image
 	selectedNodes := []clusternodes.Node{}
 	for _, node := range candidateNodes {
-		_, err := node.ImageInspect(imageID)
```

Comment:
@BenTheElder just curious, why this wasn't working? 
`crictl inspectid ImageID` returns an error if the id doesn't exist inside the image

```
# crictl inspecti 241709fdde11b50175f296c6c88c103ceb845dea69f27b350bebfc04c60a3 > /dev/null && echo "present"
present
root@kindv4-worker:/# crictl inspecti 241709fdde11b50175f296c6c88c103ceb845dea69f27b350bebfc04c60a4 > /dev/null && echo "present"
FATA[0000] no such image "241709fdde11b50175f296c6c88c103ceb845dea69f27b350bebfc04c60a4" present
root@kindv4-worker:/# crictl inspecti 241709fdde11b50175f296c6c8 > /dev/null && echo "present"
present
```

Reply:
Consider the case that two image names map to the same image.

Reply:
You need to check the (name, ID) tuple to ensure that loading would do nothing. 

We could get more clever when we observe that the ID already exists and avoid the save with some hacks but it's probably not worth the complexity and potential brittleness at the moment.

Reply:
ahh ok ,got it, I was working under the false assumption that the name -> ID relation was unique

Thanks for the explanation